### ISSUE TITLE
Fetch balance for HD: if  many addresses, only fetch latest ones (200..?)

### ISSUE THREAD

**Issue by Overtorment** [BlueWallet/BlueWallet#1547]
Should be the option to rescan the wallet from scratch somewhere in settings

**Comment by benjaminmcdonald** [comment]
Is there some incomparability with other wallets? I imported a mycelium wallet and there were some addresses missing

I'm guessing they use different HD derivative paths? Then if so can we let the user know the path and spec that bluewallet uses so a user can check compatibility

**Comment by Overtorment** [comment]
> HD derivative paths? Then if so can we let the user know the

BW by default uses paths from BIP84/BIP49/BIP44.
Im not familiar what mycelium does under the hood. for example, we were working on better BreadWallet compatibility (m/0/0), and bread at some point in the hierarchy can just switch from legacy to bech32 addresses, maybe mycelium does something similar

**Comment by ncoelho** [comment]
walletsrecovery.org

**Comment by benjaminmcdonald** [comment]
Yes in mycelium the user can switch from 
legacy to P2SH to bech32 addresses.

So a breadwallet patch with fix mycelium also

Mycelium is BIP-44 so that's already covered

**Comment by Overtorment** [comment]
No, we covered only Breadwallet format (m/0/0). Autodetection of bech32 on
legacy hierarchy (BIP44) won't work.
Unless Mycelium starts working through BIP84 for the same seed once it
decides to switch to bech32 from legacy.

**Comment by pointbiz** [comment]
Mycelium uses a single bip39 mnemonic and maps 3 derivation paths (BIP44/BIP49/BIP84) and allows the user to select per receive which format they want to use. So, they can have transactions on 1 or 2 or 3 paths from the same seed.

Mycelium is not following any standard by doing this. This undocumented non-standard is also used by Coinomi and Samourai wallet.
